Sources/Hedera/Transaction.swift

```diff
@@ -327,6 +327,8 @@ public class Transaction: ValidateChecksums {
 
         isFrozen = true
 
+        let _ = try self.getTransactionHash()
```

Comment:
it looks like you're doing this to disable transaction ID regeneration? This disables _all_ transaction ID regeneration period (there's no way to turn it back on)